web/src/js/ducks/flows.js

```diff
@@ -20,32 +23,54 @@ const defaultState = {
     view: [],
     filter: undefined,
     highlight: undefined,
+    sort: {sortColumn: undefined, sortDesc: false},
 }
 
 function makeFilterFn(filter) {
     return filter ? Filt.parse(filter) : () => true;
 }
 
+
+function makeSortFn(sort){
+    let column = _.find(flowtable_columns, c => c.name == sort.sortColumn);
+    if (!column) return;
+
+    let sortKeyFun = column.sortKeyFun;
+    if (sort.sortDesc) {
+        sortKeyFun = sortKeyFun && function () {
+            const k = column.sortKeyFun.apply(this, arguments);
```

Comment:
i copied the old implemetation, if i skip this part: 
`if (sort.sortDesc) {
        sortKeyFun = sortKeyFun && function () {
            const k = column.sortKeyFun.apply(this, arguments);
            return _.isString(k) ? reverseString("" + k) : -k;
        };
    }`
only asc sorting is possible.  what exactly shoud i replace with column.sortKeyFun(flow) ?